CMS-Enterprise/mint-app
# [MINT-3197] MTO table routing, wrapper, empty state

# MINT-3197

[Figma](https://www.figma.com/design/BcI7pNwfkmwaDD2EGGTsYC/IT-Lead?node-id=4196-76754&node-type=frame&t=pSfMcpT8jnD8BAnA-0)

## Description


- Added routing
- Added home page
  - MTO internal navigation with mobile responsiveness
  - Query, loading, error states
- Added empty state for home page
- Added `<StatusBanner />`
- Added `<OptionsPanel />`
- Unit tests


## How to test this change

1.  Create model plan
2. Navigate to `/models/modelID/collaboration-area/model-to-operation/matrix
3. Verify empty states with Figma
4. Seed database
5. Click on MTO plan
6. Navigate back to ^ route
7. Verify status banner matches

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
